react-native/src/RealmApp.js

```diff
@@ -0,0 +1,15 @@
+import {app_id, base_url} from '../realm';
+
+if (!app_id) {
+  throw 'Missing Realm App ID. Set app_id in realm.json';
+}
+if (!base_url) {
+  throw 'Missing Realm base URL. Set base_url in realm.json';
+}
+
+const appConfiguration = {
+  id: app_id,
+  baseUrl: base_url
+};
+
+export const realmApp = new Realm.App(appConfiguration);
```

Comment:
why create a realmApp export when we have the in built SDK method - https://docs.mongodb.com/realm-sdks/js/latest/Realm.App.html#getApp?

Reply:
That would require loading the configuration in various places as well. Note we're not just using the app ID.

Reply:
getApp doesn't require the configuration, just the ID. So initially you can specify a configuration that includes an app id and a base URL and then wherever you want to use the realm app, just pass the exported app id to the getApp method

Reply:
Not saying this approach doesn't work, it definitely does. I think it's a bit redundant since it accomplishes the same thing `getApp()` does, except without passing an ID. 

Especially if our goal is to make it as easy as possible to turn a scaffolded template app into their app, perhaps we don't want to include a function which they might delete later on?

Reply:
Ah that's cool. One question: how do we guarantee that any Realm.App.getApp() calls happen after the initialization with the base url?

Reply:
I see your point, getApp would create a new Realm App with just the app id, rather than throw an error and say that 'the app has not been initialized yet'